pages/04.tutorials/27.setup-jwt-authentication-with-keycloak-gatekeeper/default.en.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Setup JWT authentication with keycloak gatekeeper'
+published: true
+taxonomy:
+    tag:
+        - kubernetes
+        - Keycloak
+        - JWT
+---
+
+## Overview
+
+For applications that are API-only, setting up authentication with JWT tokens is a reasonable choice,
```

Comment:
please add an example for what scenario where you might use this to get a broader overview for whom this tutorial is for